---
geometry_msgs/PointStamped[] pois
